### PR TITLE
Add information reg. 3rd party apps to FAQ (Addresses #1461)

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR adds the information that 3rd party QR-Codes can't be scanned with the CWA currently, to the FAQ. I intentionally did not mention luca or any other brand in the hope that this won't need approval then. 

<details>
<summary>Preview of the changes</summary>

| EN | DE |
|---|---|
|Preview EN | Preview DE |

</details>

---

**More informations in #1461**

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -500,7 +500,7 @@
...
+                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Will there be a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan a Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json]:
```suggestion
                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Will there be a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",
```
Grammar correction. Please also correct Line 589 as well!

Comment (MikeMcC399) [reply src/data/faq.json]: Changes 
"What happens if I scan a Event QR code but do not have the CWA installed on my smartphone?" to
"What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?"

Comment (Ein-Tim) [reply src/data/faq.json]: Thank you! Done in https://github.com/corona-warn-app/cwa-website/pull/1482/commits/2e115eec7a5f38de455441911ceee009eddc930e.

**Review by dsarkar — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks.

**Pull request merged**